src/app/dungeon/entities/floor.component.ts

```diff
@@ -1,34 +1,28 @@
 import { ChangeDetectionStrategy, Component, computed, CUSTOM_ELEMENTS_SCHEMA, effect, input } from '@angular/core';
-import { extend } from 'angular-three';
+import { extend, NgtArgs } from 'angular-three';
 import { NgtrCuboidCollider } from 'angular-three-rapier';
-import { injectTexture } from 'angular-three-soba/loaders';
+import { textureResource } from 'angular-three-soba/loaders';
 import { Mesh, MeshBasicMaterial, NearestFilter, Object3D, PlaneGeometry, RepeatWrapping } from 'three';
 
 @Component({
   selector: 'dungeon-floor',
   template: `
-    <ngt-mesh
-      [position]="[0, 0, 0]"
-      [rotation]="[-Math.PI / 2, 0, 0]"
-      [scale]="[layout()[0].length, layout().length, 1]"
-    >
-      <ngt-plane-geometry [args]="[1, 1]" />
+    <ngt-mesh [rotation.x]="-Math.PI / 2" [scale]="[layout()[0].length, layout().length, 1]">
+      <ngt-plane-geometry *args="[1, 1]" />
```

Comment:
`*args` is different than `[args]` on the colliders but v4 should fix this confusion